### ISSUE TITLE
No package.json file

### ISSUE THREAD

**Issue by andreiropotica** [eligrey/canvas-toBlob.js#24]
I need to get this package via npm, but It cannot download because there is no package.json file.

**Comment by amandaghassaei** [comment]
https://www.npmjs.com/package/canvas-toBlob

**Comment by andreiropotica** [comment]
Yes, that is the published repo, but where is the package.json file?

(master) npm i eligrey/canvas-toBlob.js
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /Users/andreir/.npm/_cacache/tmp/git-clone-469bc4e4/package.json
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/Users/andreir/.npm/_cacache/tmp/git-clone-469bc4e4/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/andreir/.npm/_logs/2022-11-05T16_28_36_939Z-debug.log
